assembl/models/user_key_values.py

```diff
@@ -232,6 +243,86 @@ def __contains__(self, key):
         return value is not None
 
 
+class NamespacedKVCollection(MutableMapping):
```

Comment:
@ayyazdaniaryan : All accurate. Added some minimal comments.
As a matter of fact, I meant to store many things in IdeaNamespacedKeyValue instead of column objects, but as the list of properties got longer, I created the column objects and it no longer made sense.